upload/library/Sidane/Threadmarks/Install.php

```diff
@@ -48,9 +48,9 @@ public static function install($existingAddOn, $addOnData)
           $db->query("ALTER TABLE threadmarks add unique index thread_post_id (`thread_id`,`post_id`)");
       }
 
-      $db->query("insert ignore into xf_permission_entry_content (content_type, content_id, user_group_id, user_id, permission_group_id, permission_id, permission_value, permission_value_int)
-        select distinct content_type, content_id, user_group_id, user_id, convert(permission_group_id using utf8), 'sidane_tm_manage', permission_value, permission_value_int
-        from xf_permission_entry_content
+      $db->query("insert ignore into xf_permission_entry (content_type, content_id, user_group_id, user_id, permission_group_id, permission_id, permission_value, permission_value_int)
+        select distinct user_group_id, user_id, convert(permission_group_id using utf8), 'sidane_tm_manage', permission_value, permission_value_int
+        from xf_permission_entry
         where permission_group_id = 'forum' and permission_id in ('warn','editAnyPost','deleteAnyPost')
         ");
```

Comment:
Error when installing:

    Server Error
    Mysqli statement execute error : Column count doesn't match value count at row 1

    .../Sidane/Threadmarks/Install.php at line 55

Are you testing these pull requests locally before submitting them? I've got a fresh XenForo install with this as the only add-on. When testing new features/pull requests I uninstall the threadmark and reinstall with the new version.

If you're not doing that already, it's definitely worthwhile as it will catch bugs like this (and quite a few that have come before).

Cheers.